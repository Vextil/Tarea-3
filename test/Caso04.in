# ColaBinario
Desencolar
CrearBinario martes 5 1 dia 4 gran 4 altura 2 tambien 9 zoo 12 water 43 ya 12 FIN
Izquierdo 1 2
Derecho 1 3
Izquierdo 2 4
Derecho 2 5
Derecho 3 6
FiltroBinario 1 2 < 7
Encolar 1
Desencolar
Desencolar
Encolar 1
Encolar 2
Encolar 3
Encolar 4
Encolar 5
Encolar 6
Encolar 7
Desencolar
Desencolar
Desencolar
Desencolar
Desencolar
Desencolar
Desencolar
Fin
