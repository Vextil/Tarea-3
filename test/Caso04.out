1># ColaBinario
2>La cola estaba vacia.
3>OK
4>OK
5>OK
6>OK
7>OK
8>OK
9>OK
10>OK
11>
   (12,zoo)
     (12,ya)
    (43,water)
  (9,tambien)
 (5,martes)
   (4,gran)
  (4,dia)
   (2,altura)

12>La cola estaba vacia.
13>OK
14>OK
15>OK
16>OK
17>OK
18>OK
19>OK
20>
   (12,zoo)
     (12,ya)
    (43,water)
  (9,tambien)
 (5,martes)
   (4,gran)
  (4,dia)
   (2,altura)

21>
  (4,gran)
 (4,dia)
  (2,altura)

22>
  (12,zoo)
    (12,ya)
   (43,water)
 (9,tambien)

23>
 (2,altura)

24>
 (4,gran)

25>
 (12,zoo)
   (12,ya)
  (43,water)

26>
   (12,zoo)
     (12,ya)
    (43,water)
  (9,tambien)
 (5,martes)
   (4,gran)
  (4,dia)

27>Fin
